doajtest/unit/test_api_bulk_application.py

```diff
@@ -185,7 +185,7 @@ def test_05_test_via_endpoint(self):
                 first_apl = reply.pop()
                 assert first_apl['status'] == 'created'
                 # Check we actually created new records
-                time.sleep(1)
+                time.sleep(1.5)
```

Comment:
I guess you did this because you were having problems getting the test to pass, due to the race condition here.  In this instance I think this is fine - the test doesn't give you much alternative, but just to note for information that eventually we want to squeeze all these `time.sleep` calls out.

Reply:
Yup, that was me, seemed to help codeship. Agreed on the `time.sleep`s